core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -21,6 +21,8 @@ trait RawlsInstrumented extends WorkbenchInstrumented {
   final val SubsystemMetricKey         = "subsystem"
   final val WorkflowStatusMetricKey    = "workflowStatus"
   final val WorkspaceMetricKey         = "workspace"
+  final val StatsBoardMetricKey        = "statsBoard"
+  final val RawlsBoardMetricValue      = "rawlsBoard"
```

Comment:
I don't think we should tie the metric paths to the dashboard they appear in, since we could always change dashboards. I would vote to group them logically based on function, so something like "workspaceAggregates" as a metric key